From 5b0ddc879f7962e8c2ac73d248a93282a152b6e3 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler <raster@openmoko.org>
Date: Wed, 29 Oct 2008 10:36:47 +1100
Subject: [PATCH] Exqusitie: Stay around for longer if the boot is very long

Increase the timeout for exqusitie from 180 to 300 seconds in case the boot is
(legitmately) long. Also up the delay in waiting for exqusitie to start to 40
seconds for the same reasons.
---
 packages/e17/exquisite/exquisite-init |    6 +++---
 packages/e17/exquisite_svn.bb         |    2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/packages/e17/exquisite/exquisite-init b/packages/e17/exquisite/exquisite-init
index 60ee13f..a504ee1 100644
--- a/packages/e17/exquisite/exquisite-init
+++ b/packages/e17/exquisite/exquisite-init
@@ -33,8 +33,8 @@ export EXQUISITE_IPC=$EXDIR/exquisite
 mount tmpfs -t tmpfs $EXDIR -o,size=40k
 exquisite -fb -fs $args &
 
-exquisite-write -wait 20
+exquisite-write -wait 40
 exquisite-write "PULSATE"
 
-# kill off exquisite in 180 seconds if its not gone yet
-(sleep 180; exquisite-write "QUIT")&
+# kill off exquisite in 300 seconds if its not gone yet
+(sleep 300; exquisite-write "QUIT")&
diff --git a/packages/e17/exquisite_svn.bb b/packages/e17/exquisite_svn.bb
index 775b280..218a99a 100644
--- a/packages/e17/exquisite_svn.bb
+++ b/packages/e17/exquisite_svn.bb
@@ -2,7 +2,7 @@ DESCRIPTION = "A psplash replacement for display"
 LICENSE = "MIT BSD"
 DEPENDS = "eet evas ecore embryo edje"
 PV = "0.0.1+svnr${SRCREV}"
-FILE_PR = "r2"
+FILE_PR = "r3"
 RDEPENDS = "initscripts"
 RRECOMMENDS_${PN} = "exquisite-themes"
 
-- 
1.5.4.3

